### ISSUE TITLE
Certus quartz in AE increases by 38000

### ISSUE THREAD

**Issue by MaucnAD** [NillerMedDild/Enigmatica2Expert#848]
**Modpack version**
1.55

**Can the issue be reproduced?**
unknown

**Issue**
The certus quartz in my AE system doesn't increase by the number I put in (e.g. +1 certus, +2 certus, +64 certus), instead it increases by 38000 certus quartz.  I have no idea how this can happen. Obviously I have not tried it with most items, but it seems to only happen to certus quartz. 

They are not phantom items as the number DOES goes down when I pull out some certus, and my drives are full of certus quartz now...

**Comment by NillerMedDild** [comment]
What the actual fuck :P

**Comment by MaxNeedsSnacks** [comment]
I tried reproducing that glitch ingame with all different drive sizes and had no luck in doing so unfortunately.

Could you try getting ALL of the certus out of the system, removing all drives but one and then inserting some of it back into the system (with multiple drives if possible).

I haven't had the time to test *that* specific scenario yet

**Comment by MaucnAD** [comment]
> I tried reproducing that glitch ingame with all different drive sizes and had no luck in doing so unfortunately.
> 
> Could you try getting ALL of the certus out of the system, removing all drives but one and then inserting some of it back into the system (with multiple drives if possible).
> 
> I haven't had the time to test _that_ specific scenario yet

Found something out, I have most of my mining resources (including certus quartz) in my drawers with a storage bus connected to it, when my drawers aren't connected to the system everything is normal, but when the certus gets recognized, it goes up by 38000. This only happens to certus though, which is weird.

**Comment by NillerMedDild** [comment]
How is the Certus Quartz stored in your drawer system?

**Comment by MaucnAD** [comment]
with a compacting drawer

**Comment by NillerMedDild** [comment]
Thought so. Maybe the Compacting Drawer thinks 1 Certus Quartz Block = 9 Certus Quartz, but 9 Certus Quartz = 2 Certus Quartz Blocks. Maybe it's self-duping :D

**Comment by MaucnAD** [comment]
Makes sense, but why would it still dupe the certus after the drawer is full? Wouldn't it just work like a normal AE system?

**Comment by MaxNeedsSnacks** [comment]
As you said in your OP, the drawer doesn't actually dupe the items, it just SHOWS the duped items where there are none (if our theory is to be trusted). The Certus block it's converting from then would be JAOPCA's, while the block it's actually outputting to would be AE2's quartz block

**Comment by NillerMedDild** [comment]
I disabled the one from JAOPCA though.

**Comment by MaxNeedsSnacks** [comment]
Huh... weird. My suggestion (and what I ended up actually doing until I ditched my drawer system completely) for now would be to just use a regular 1x1 drawer. I'm gonna do some more testing later on

**Comment by MorganSimic** [comment]
I heard some people having issues with certus quartz compacting. Maybe something with the jaopca removal made it weird. I will also test it later today.

**Comment by NillerMedDild** [comment]
I can't reproduce the issue.

**Comment by zapl** [comment]
I also had a sudden overflow in my system, there were several million certus quartz filling all my drives. I too had some of it in compacting drawers connected via storage bus and the drawer was probably full at some point. It also behaved strange: the compacting drawer from 1.54 showed certus quartz in the top slot while a new one I set up showed it bottom left and a certus block correctly in the top.
Maybe the issue is that when AE tries to insert a few certus into a broken the drawer system (via storage bus for me), the remainder after inserting is reported as 38k certus and AE then puts those into it's disk.

**Comment by MaucnAD** [comment]
> I also had a sudden overflow in my system, there were several million certus quartz filling all my drives. I too had some of it in compacting drawers connected via storage bus and the drawer was probably full at some point. It also behaved strange: the compacting drawer from 1.54 showed certus quartz in the top slot while a new one I set up in 1.55 showed it bottom left and a certus block correctly in the top.
> Maybe the issue is that when AE tries to insert a few certus into a broken drawer system, the remainder after inserting is reported as 38k certus and AE then puts those into it's disk.
> Issue is gone for me after I moved the certus into a black hole unit and broke the misbehaving compacting drawer, but there's still some in the new compacting drawer.

Whatever it was, the update fixed it for me. Still storing it in a compactring drawer.

**Comment by Wntermute** [comment]
Is this how I suddenly have 139M Certus Quartz and filled *NINE* 16384K disks?

**Comment by NillerMedDild** [comment]
Yes! Congrats! :P

**Comment by Wntermute** [comment]
No worries.. More silicon to make more disks

**Comment by AxxiD** [comment]
I just had the same issue after updating my server.
I updated my server from 1.49 to 1.58 and starting the server on every version in between.
After the last update we played for a bit and we suddenly had 2.5 billion Certus Quartz and growing rapidly (like 3 million every 5 seconds or so which is alot more than what we usually get from our mystical crops).
We found this issue and searched for any compacting drawer containing certus and removed it from our network. After that the duplicating stopped.

We for our part just won't use them again for certus.
